src/ome_zarr_models/_rfc5_transforms/coordinate_transformations.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from typing import Annotated, Literal
+
+from pydantic import Field, field_validator
+
+from ome_zarr_models._rfc5_transforms.axes import Axes
+from ome_zarr_models._utils import duplicates
+from ome_zarr_models.base import BaseAttrs
+
+
+class CoordinateSystem(BaseAttrs):
+    name: str
+    axes: Axes
+
+    @field_validator("name")
+    def name_must_not_be_empty(cls, value: str) -> str:
```

Comment:
Can you just use a `Field(..., min_length=1)` above, instead of defining this custom validator?

Reply:
👍